### ISSUE TITLE
recreating homo_search.py output -- minimal version

### ISSUE THREAD

**Issue by avilella** [dptech-corp/Uni-Fold#135]
Hi,

I am running Uni-Fold on antibody-antigen pairs, where the antigen (chain A) is always the same, and the antibody sequences (chain B in each prediction) are very similar to each other (same species).

Since the `homo_search.py` part of run_unifold.sh multimer takes a long time, but produces very similar hits, I would like to recreate it in a new folder for new predictions, so I can just calculate the second `inference.py` part of `run_unifold.sh` on it.

My plan is to aggregate each of the `.sto` files for a bunch of predictions, and produce a combined version in the new input folder structure to `inference.py`. The `.sto` format is a bit cumbersome to recreate, and if the `inference.py` part is not going to read the alignment structure from it, but rather just the fasta entries, would it be possible to provide the "combined inputs" as multi-fasta files rather than `.sto` files?

Thanks in advance.

**Comment by ZiyaoLi** [comment]
I would recommend you to refer to the mmseqs processing code [here](https://github.com/dptech-corp/Uni-Fold/blob/main/unifold/colab/mmseqs.py) and [here](https://github.com/dptech-corp/Uni-Fold/blob/595c0c3a3295c2fc8933d0d8473e64ce917c1904/notebooks/unifold.ipynb#L210). It has a lighter processing pipeline.

**Comment by avilella** [comment]
If I am reading the code in `inference.py` correctly, for multimer, it reads the uniprot_msa_dir?

```
def load_feature_for_one_target(
    config, data_folder, seed=0, is_multimer=False, use_uniprot=False
):
    if not is_multimer:
        uniprot_msa_dir = None
        sequence_ids = ["A"]
        if use_uniprot:
            uniprot_msa_dir = data_folder

    else:
        uniprot_msa_dir = data_folder
        sequence_ids = open(os.path.join(data_folder, "chains.txt")).readline().split()
    batch, _ = load_and_process(
        config=config.data,
        mode="predict",
        seed=seed,
        batch_idx=None,
        data_idx=0,
        is_distillation=False,
        sequence_ids=sequence_ids,
        monomer_feature_dir=data_folder,
        uniprot_msa_dir=uniprot_msa_dir,
        is_monomer=(not is_multimer),
    )
    batch = UnifoldDataset.collater([batch])
    return batch


def main(args):
    config = model_config(args.model_name)
    config.data.common.max_recycling_iters = args.max_recycling_iters
-UU-:----F1  inference.py   13% (70,0)    Git-main  (Python ElDoc) ---------------------------------------------------------------------------------------------------------------------------------------------------
```

**Comment by ZiyaoLi** [comment]
Yes. Uniprot msas are used for msa-pairing because they contain species information.
